src/pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>de.terrestris</groupId>
     <artifactId>shogun2</artifactId>
-    <version>1.0.5</version>
```

Comment:
Is this a problem? It seems that there are two commits missing (after the last release to set the new snapshot version)

Reply:
Well, I don't know where those commits are. We didn't have a `1.0.x` branch yet, so I created one from the `1.0.5` tag, hence the custom version update.

Reply:
ok